packages/metal-quartz-button/src/MetalQuartzButtonAttributes.soy

```diff
@@ -0,0 +1,36 @@
+{namespace MetalQuartzButtonAttributes}
+
+/**
+ * This renders the component's whole content.
+ */
+{template .attributes kind="attributes"}
+	{@param? block: bool}
+	{@param? disabled: bool}
+	{@param? name: string}
+	{@param? size: string}
+	{@param? style: string}
+	{@param? type: string}
+	{@param? value: string}
+
+	{let $blockClass: $block ? ' btn-block' : '' /}
+	{let $sizeClass: $size ? ' btn' + $size : '' /}
+	{let $styleClass: $style ? ' btn-' + $style : ' btn-default' /}
+
+	class="btn{$blockClass}{$styleClass}{$sizeClass}"
```

Comment:
In fact the Incremental DOM compiler is complaining. Let's try to think of something.